dashboard/app/models/user.rb

```diff
@@ -194,11 +194,12 @@ def school_info_optional?
 
   belongs_to :invited_by, polymorphic: true
 
-  validate :admins_must_be_teachers
+  validate :admins_must_be_teachers_without_followeds
```

Comment:
Very true, I'm simply following existing "spelling" convention. Doubtful that I'll prioritize fixing this across our codebase, though I'll keep it in mind.